From 0628fd53b04714b1b701251ce3626ab965e3d592 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 30 Oct 2017 13:03:56 -0500
Subject: [PATCH] Suppress HealthCheck for test_batch_outer_computes_correctly

=================================== FAILURES ===================================
_____________________ test_batch_outer_computes_correctly ______________________

cpu_ops = <thinc.neural.ops.NumpyOps object at 0x115d30a10>

    @settings(max_examples=MAX_EXAMPLES)
>   @given(arrays_BI_BO=strategies.arrays_BI_BO())
    def test_batch_outer_computes_correctly(cpu_ops, arrays_BI_BO):
E   FailedHealthCheck: Data generation is extremely slow: Only produced 9 valid
examples in 1.08 seconds (0 invalid ones and 2 exceeded maximum size). Try
decreasing size of the data you're generating (with e.g.average_size or
max_leaves parameters).
---
 thinc/tests/unit/test_ops.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/thinc/tests/unit/test_ops.py b/thinc/tests/unit/test_ops.py
index cdc880a..6397690 100644
--- a/thinc/tests/unit/test_ops.py
+++ b/thinc/tests/unit/test_ops.py
@@ -1,6 +1,6 @@
 import pytest
 import numpy
-from hypothesis import given, settings
+from hypothesis import given, settings, HealthCheck
 from numpy.testing import assert_allclose
 
 from .. import strategies
@@ -144,7 +144,7 @@ def test_batch_dot_computes_correctly(cpu_ops, W_b_inputs):
     assert_allclose(y, expected)
 
 
-@settings(max_examples=MAX_EXAMPLES)
+@settings(max_examples=MAX_EXAMPLES, suppress_health_check=[HealthCheck.too_slow])
 @given(arrays_BI_BO=strategies.arrays_BI_BO())
 def test_batch_outer_computes_correctly(cpu_ops, arrays_BI_BO):
     bi, bo = arrays_BI_BO
-- 
2.14.1

